### DISCUSSION TITLE
Analyzing Takara's newest kit: SMART-Seq Mouse BCR (with UMIs)

### DISCUSSION THREAD

**Discussion by PaulDanPhillips** [milaboratory/mixcr#1563] [Q&A] [answered]
I appreciate that this kit just came out from Takara, but I'm hoping you can help me best run mixcr on sequences from this new kit. 

1. Would you recommend running the available mixcr preset for humans and just overriding the reference species (shown below)\
`mixcr analyze takara-human-rna-bcr-umi-smartseq -s mouse input_R1.fastq.gz input_R2.fastq.gz result 
`
2. What other recommendations would you make to best analyze data from Takaras SMART-Seq Mouse BCR (with UMIs)?
3. When do you anticipate a new preset will be available within mixcr for this kit?

Thanks!

**Comment by mizraelson** [comment, marked as answer]
Hi, thanks for pointing the new preset out. I will add it in 1-2 days. Meanwhile you can use the command you have shared:
```
mixcr analyze takara-human-rna-bcr-umi-smartseq \
-s mouse \
input_R1.fastq.gz \
input_R2.fastq.gz \
result
```

It should work exactly like the new preset will.
